.buildkite/pipeline.elastic-agent-package.yml

```diff
@@ -76,7 +77,12 @@ steps:
           .buildkite/scripts/steps/package.sh
         artifact_paths:
           - "build/distributions/**/*"
-      - label: ":package: Package ARM elastic-agent"
+        matrix:
+          setup:
+            fips:
+              - false
+              - true
+      - label: ":package: FIPS={{matrix.fips}} Package ARM elastic-agent"
         key: package_elastic-agent-arm
```

Comment:
I don't know if `package_elastic-agent-arm` will cause any issues with a matrix, and whether it should be unique here :/